test/gtest/uct/test_md.cc

```diff
@@ -731,10 +762,12 @@ UCS_TEST_SKIP_COND_P(test_md, dereg_bad_arg,
     uct_mem_h memh;
     void *ptr;
     ucs_status_t status;
+    uct_md_mem_reg_params_t reg_params;
     uct_md_mem_dereg_params_t params;
 
-    ptr    = malloc(size);
-    status = uct_md_mem_reg(md(), ptr, size, md_flags, &memh);
+    ptr = malloc(size);
+    set_reg_params(&reg_params, md_flags, ptr, size);
+    status = uct_md_mem_reg_v2(md(), &reg_params, &memh);
```

Comment:
maybe wrap both functions?
reg_mem(ptr, size, md_flags) that also calls uct_md_mem_reg_v2()